addons/jump-to-def/addon.json

```
@@ -19,6 +19,10 @@
     }
   ],
   "versionAdded": "1.28.0",
+  "latestUpdate": {
+    "version": "1.41.0",
+    "temporaryNotice": "You can now use the context menu (open by right clicking or touching and holding) instead of the keyboard shortcut to jump to custom block definitions."
```

```suggestion
    "temporaryNotice": "You can now use the context menu (open by right clicking or touching and holding) to jump to custom block definitions."
```
I think it should be clear that the old way to go to the definition isn't being removed.